### PR TITLE
chore: add catalog-info.yaml

### PR DESCRIPTION
**Pull request by Magnet-js**
### Type of Change

- [ ] Bugfix
- [ ] Enhancement / new feature
- [ ] Refactoring
- [ ] Documentation

### Description

Added catalog-info.yaml because it's a required file
(for more info [backstage](https://backstage.staffbase.com/docs/default/component/advisory-space/adrs/principles-processes/0005-required-github-files/))

### Checklist

- [ ] Write tests
- [ ] Make sure all tests pass
- [ ] Update documentation
- [ ] Review the [Contributing Guideline](https://github.com/Staffbase/yamllint-action/blob/main/CONTRIBUTING.md) and sign CLA
- [ ] Reference relevant issue(s) and close them after merging

### CODE REVIEW

File: catalog-info.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: autodev-action
+  annotations:
+    github.com/project-slug: Staffbase/autodev-action
+    backstage.io/source-location: url:https://github.com/Staffbase/autodev-action
+    backstage.io/techdocs-ref: dir:.
```
Comment (axdotl) [inline catalog-info.yaml:8]: This repo does not have any TechDocs, yet.
```suggestion
```

File: catalog-info.yaml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  tags:
+    - typescript
```
Comment (axdotl) [inline catalog-info.yaml:10]:
```suggestion
    - github-action
    - typescript
```

**Review by axdotl — CHANGES_REQUESTED** [review]

File: catalog-info.yaml [hunk 3]
```diff
@@ -0,0 +1,14 @@
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: autodev-action
```
Comment (axdotl) [inline catalog-info.yaml:4]: A description would be nice
```suggestion
  name: autodev-action
  description: This GitHub action merges labeled PRs into the dev branch.
```

File: catalog-info.yaml [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+  description: This GitHub action merges labeled PRs into the dev branch.
+  annotations:
+    github.com/project-slug: Staffbase/autodev-action
+  tags:
+    - github-action
+    - typescript
+spec:
+  type: other
+  lifecycle: production
```
Comment (axdotl) [inline catalog-info.yaml:13]: Please add missing new-line to end of file

```suggestion
  lifecycle: production

```

**Review by staffbase-robert — APPROVED** [review]

**Review by 0x46616c6b — APPROVED** [review]

**Pull request merged**